python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -870,6 +870,11 @@ def test_agg_with_avg(self):
         self.assertEqual(4.0, res[0][1])
         self.assertEqual(5.0, res[1][1])
 
+    def test_column_operator(self):
```

Comment:
nit: maybe we should move all such tests to `test_connect_functions.py`?

Reply:
or create one `test_connect_columns.py`